setup/azurecmdfiles.wxi

```diff
@@ -968,6 +962,9 @@
                         <Component Id="cmp33ACC536D3148134079F201DDFE77041" Guid="*">
                             <File Id="fil68134395180424CF926E86CA852D5A17" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.DataLakeAnalytics\Microsoft.Rest.ClientRuntime.Azure.Authentication.dll" />
                         </Component>
+                        <Component Id="cmp7AF20CAE2005123323E3D9C0F8709408" Guid="*">
+                            <File Id="filBD917B281F92F71FE5811C63D626F43C" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.DataLakeAnalytics\Microsoft.Rest.ClientRuntime.Azure.dll" />
```

Comment:
?? Where does this reference come from?  My guess is that this will cause the build to fail